README.md

```diff
@@ -23,6 +23,9 @@ OpenSearch Java Client
 For more information, see [opensearch.org](https://opensearch.org/).
 This client is meant to replace the existing [OpenSearch Java High Level REST Client](https://opensearch.org/docs/latest/clients/java-rest-high-level/).
 
+## Sample code
```

Comment:
Add to TOC as well, please.

Reply:
Done. Sorry about so many commits. Hope you'll squash and merge :)